sources/views/form.js

```diff
@@ -94,8 +109,12 @@ export default class FormView extends JetView {
         const form = this.$$("contactForm");
 
         this.on(this.app, "onCancel", () => {
-            const preDvata = form.prevData;
-            if (preDvata) form.setValues(preDvata);
+            if (form.isDirty()) {
```

Comment:
this is checked before the popup is shown, so this event never fires if form is not dirty. so here it's not necessary

Reply:
also you probably do not need to check if clean values exist since this code runs if the form is dirty so clean values exist. just set them